contracts/payloads/IGP83/PayloadIGP83.sol

```diff
@@ -0,0 +1,734 @@
+pragma solidity ^0.8.21;
+pragma experimental ABIEncoderV2;
+
+import {BigMathMinified} from "../libraries/bigMathMinified.sol";
+import {LiquidityCalcs} from "../libraries/liquidityCalcs.sol";
+import {LiquiditySlotsLink} from "../libraries/liquiditySlotsLink.sol";
+
+import {IGovernorBravo} from "../common/interfaces/IGovernorBravo.sol";
+import {ITimelock} from "../common/interfaces/ITimelock.sol";
+
+import {IFluidLiquidityAdmin, AdminModuleStructs as FluidLiquidityAdminStructs} from "../common/interfaces/IFluidLiquidity.sol";
+import {IFluidReserveContract} from "../common/interfaces/IFluidReserveContract.sol";
+
+import {IFluidVaultFactory} from "../common/interfaces/IFluidVaultFactory.sol";
+import {IFluidDexFactory} from "../common/interfaces/IFluidDexFactory.sol";
+
+import {IFluidDex, IFluidAdminDex, IFluidDexResolver} from "../common/interfaces/IFluidDex.sol";
+
+import {IFluidVault, IFluidVaultT1} from "../common/interfaces/IFluidVault.sol";
+
+import {IFTokenAdmin, ILendingRewards} from "../common/interfaces/IFToken.sol";
+
+import {IDSAV2} from "../common/interfaces/IDSA.sol";
+import {IERC20} from "../common/interfaces/IERC20.sol";
+import {IProxy} from "../common/interfaces/IProxy.sol";
+import {PayloadIGPConstants} from "../common/constants.sol";
+import {PayloadIGPHelpers} from "../common/helpers.sol";
+
+contract PayloadIGP83 is PayloadIGPConstants, PayloadIGPHelpers {
+    uint256 public constant PROPOSAL_ID = 83;
+
+    // New state variables for delay
+    uint256 public delayTime;
+    uint256 public delaySetTime;
+
+    mapping(uint256 => bool) public setActionSkippable;
+
+    modifier isSkippable(uint256 action_) {
+        // If function is not skippable, then execute
+        if (setActionSkippable[action_] == false) {
+            _;
+        }
+    }
```

Comment:
@thrilok209 This won't work as expected right? We need a revert or something here.

Reply:
@pradyuman-verma Why revert?

Reply:
Shriya updated this logic
i think @thrilok209 has already guided her to create this though maybe he can reconfirm

Reply:
I tested it in remix, tx never reverts.

Reply:
@pradyuman-verma shouldn't revert, should just skip the action